content/applications/essentials/contacts.rst

```diff
@@ -178,13 +178,64 @@ section, and other partner options, including :guilabel:`Partner Activation` and
 Review` configurations. These are **only** present when the *Enterprise Resellers* module is
 installed.
 
+To publish a list of all active partners on the website, a new webpage must be created first.
+Navigate to :menuselection:`Website app --> Site --> Pages` and click :guilabel:`New`. Hover over
+the :guilabel:`Blank Page` template, and click the :guilabel:`Use this template` button to create a
+new page.
+
+In the :guilabel:`New Page` pop-up window, enter `Partners` in the :guilabel:`Page Title` field.
+Then, click :guilabel:`Create`. In the :guilabel:`Edit` toolbar on the far-right of the page, click
+:guilabel:`Save`.
```

Comment:
Do you think it would be beneficial to briefly allude to how the user can make any necessary edits/adjustments to the page, if desired, from this toolbar? Or, do you think that is already implied, and they can just click 'Save' to move along with the process?

Reply:
I don't want to overstep too much into the web builder process, especially because I think this doc is already overly arduous